env-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+REPOBASE=$(realpath $(dirname "$0"))
+LIRIDIR="REPOBASE/.build"
```

Comment:
why .build instead of ~/.opt/liri like in the docs?

Reply:
Firstly, because it's a script. I would expect from a script inside the repo to stay inside the repo. Secondly, because build artifacts can get quite expensive in terms of disk size, I would rather have them contained inside the repo so that 3. If disk space is rare or for whatever other reason I would like to get rid of liri again, I would expect to just remove the repo and am fine.

We don't have 25gb of sources and 15gb of generated build artifacts, but still, I consider this the way to go

Reply:
What about making it configurable with an argument and be $REPOBASE/.build by default.
Oh I just noticed there's a typo it should be `LIRIDIR="$REPOBASE/.build"`